pkgs/development/libraries/exiv2/default.nix

```diff
@@ -15,35 +15,35 @@
 
 stdenv.mkDerivation rec {
   pname = "exiv2";
-  version = "0.27.2";
+  version = "0.27.3";
 
   src = fetchFromGitHub {
     owner = "exiv2";
     repo  = "exiv2";
     rev = "v${version}";
-    sha256 = "0n8il52yzbmvbkryrl8waz7hd9a2fdkw8zsrmhyh63jlvmmc31gf";
+    sha256 = "0d294yhcdw8ziybyd4rp5hzwknzik2sm0cz60ff7fljacv75bjpy";
   };
 
-  patches = [
-    # included in next release
-    (fetchpatch {
-      name = "cve-2019-20421.patch";
-      url = "https://github.com/Exiv2/exiv2/commit/a82098f4f90cd86297131b5663c3dec6a34470e8.patch";
-      sha256 = "16r19qb9l5j43ixm5jqid9sdv5brlkk1wq0w79rm5agxq4kblfyc";
-      excludes = [ "tests/bugfixes/github/test_issue_1011.py" "test/data/Jp2Image_readMetadata_loop.poc" ];
-    })
-  ];
-
   cmakeFlags = [
     "-DEXIV2_BUILD_PO=ON"
     "-DEXIV2_BUILD_DOC=ON"
     # the cmake package does not handle absolute CMAKE_INSTALL_INCLUDEDIR correctly
     # (setting it to an absolute path causes include files to go to $out/$out/include,
     #  because the absolute path is interpreted with root at $out).
+    # Can probably be removed once https://github.com/Exiv2/exiv2/pull/1263 is merged.
     "-DCMAKE_INSTALL_INCLUDEDIR=include"
     "-DCMAKE_INSTALL_LIBDIR=lib"
   ];
 
+  patches = [
```

Comment:
Could you keep this just below src, see https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887.

Reply:
Hmm, though it does not seem to work here. Maybe we also need to apply https://github.com/Exiv2/exiv2/commit/dd2d181755a6e642c0a8e3225ef5407fff49eb3a

Reply:
You want to keep the other patch too, or just the one you're proposing?

Reply:
Yes, the conflict would need to be resolved.

Reply:
I'm not sure how to resolve the conflict and I don't have any arm systems to build against.

Reply:
I have resolved the conflict in https://github.com/Exiv2/exiv2/pull/1271.

Currently verifying it natively on https://github.com/nix-community/aarch64-build-box (`nix-build -A exiv2.all --argstr system aarch64-linux`) and locally by cross-compiling (`nix-shell -A pkgsCross.aarch64-multiplatform.netpbm`).

Reply:
@jtcoolen looks good!